adagrams/game.py

```diff
@@ -1,11 +1,113 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
 def draw_letters():
-    pass
+    new_pool = []
+    for letter, count in LETTER_POOL.items():
+        new_pool.extend([letter] * count)
+    
+    letters = []
+    while len(letters) < 10:
+        draw = random.randrange(0, len(new_pool))
+        letter = new_pool.pop(draw)
+        letters.append(letter)
+
+    return letters
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    uppercase_word = word.upper()
+    bank_copy = list(letter_bank)
+    
+    for char in uppercase_word:
+        if char in bank_copy:
+            bank_copy.remove(char)
+        else:
+            return False
+    return True
+
 
 def score_word(word):
-    pass
+    total_score = 0
+    for char in word.upper():
+        total_score += SCORE_CHART[char]
+    if len(word) > 6:
+        total_score += 8
+    return total_score
+
+def wins_tie(word, winner):
```

Comment:
Nice job putting the tie logic in a helper function. 

I was a little slow to understand how this function works because I was confused by the variable names. I realized that you're comparing `word` (the subsequent highest scoring word) against `winner` (the previous highest scoring word). 

I think wins_ties() could be renamed to something like current_word_wins_ties() and the params could be current_word and previous_winner.